rust/kernel/prelude.rs

```diff
@@ -15,4 +15,12 @@ pub use alloc::{borrow::ToOwned, string::String};
 
 pub use module::module;
 
-pub use super::{println, static_assert, KernelModule, KernelResult};
+pub use super::{alert, cont, crit, devel, emerg, err, info, notice, warn};
+
+pub use super::{
+    pr_alert, pr_cont, pr_crit, pr_devel, pr_emerg, pr_err, pr_info, pr_notice, pr_warn,
```

Comment:
Are these functions expected to be called directly or only through macros? If the latter, I think it makes more sense to leave them out of here and prefix their usage in macros with $crate, this way we don't avoid polluting the global namespace.

Reply:
These are macros, and they are the entry point users are expected to use to print their messages: e.g. see `rust_example.rs`.

If it was otherwise, yes, completely agreed.

Reply:
Ah, I was confused by the two versions `x` and `pr_x`. Makes sense.